### PR TITLE
Add: Project Spotify Clone to the repository.

### PR DESCRIPTION
**Pull request by vishnu-mouli-102408**

### CODE REVIEW

File: src/DB/projects.json [hunk 1]
```diff
@@ -265,5 +265,12 @@
     "title": "Movie Trailer",
     "description": "The movie-trailer app is a web application that allows users to search and view trailers for their favorite movies. The application is developed using React and integrated with the TMD and YouTube APIs to fetch movie trailers. The application has a responsive design that adapts to different screen sizes and features a search bar and filter option to help users navigate through the content.",
     "tech": ["React", "CSS", "JavaScript", "HTML", "TMD API", "Youtube API"]
+  },
+  {
+    "gh-username": "vishnu-mouli-102408",
+    "link": "spotify-clone-102408.netlify.app",
```
Comment (priyankarpal) [inline src/DB/projects.json:271]: add a valid link. it's not working.

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/DB/projects.json (hunk 1 above)
Comment (vishnu-mouli-102408) [reply src/DB/projects.json:271]: Bro it's working. I tried it now.plz tell me which error it's showing.so that I can resolve it.

Comment (priyankarpal) [reply src/DB/projects.json:271]: you can add github link

Comment (vishnu-mouli-102408) [reply src/DB/projects.json]: I changed it bro.

**Comment by priyankarpal** [comment]
@vishnu-mouli-102408 I would say that please check the docs one more time & create a new PR we added new features. so there are some conflict errors. https://github.com/priyankarpal/ProjectsHut/blob/main/contributing.md

**Comment by vishnu-mouli-102408** [comment]
Okay

**Comment by priyankarpal** [comment]
> Okay

so You can close this PR. After that create an issue for project addition. create the PR after when we will assign you

**Pull request closed without merging**